Rakefile

```diff
@@ -18,7 +18,7 @@ task :new_cop, [:cop] do |_task, args|
   require "rubocop"
 
   cop_name = args.fetch(:cop) do
-    warn("usage: bundle exec rake new_cop[Department/Name]")
+    warn('usage: bundle exec rake "new_cop[Department/Name]"')
```

Comment:
Just in case we need to revert the current change, do you mind moving this to a separate PR please 🙏